ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/toolbar/DiskResourceViewToolbarImpl.java

```diff
@@ -266,9 +266,12 @@ public void onDiskResourceSelectionChanged(DiskResourceSelectionChangedEvent eve
         final boolean isSelectionEmpty = selectedDiskResources.isEmpty();
         final boolean isSingleSelection = selectedDiskResources.size() == 1;
         final boolean isOwner = isOwnerList(selectedDiskResources);
+        final boolean isWriteable = isWritable();
+        DiskResource first_item = getFirstDiskResource();
```

Comment:
This `first_item` variable should probably be camel-cased like the others: `firstItem`